src/tqec/templates/base.py

```diff
@@ -57,10 +62,18 @@ def __init__(self, shape: BaseShape) -> None:
         Each template should have a shape, represented by a Shape instance, and encoding
         how the template scales and its plaquettes.
 
+        The default increments define the distance between two plaquettes.
+        For example a default_x_increment of 2 means that two 2x2 plaquettes will share
+        a common edge.
+
         :param shape: the underlying template shape.
+        :param default_x_increment: default increment in the x direction between two plaquettes.
+        :param default_y_increment: default increment in the y direction between two plaquettes.
         """
         super().__init__()
         self._shape_instance = shape
+        self._default_x_increment = default_x_increment
+        self._default_y_increment = default_y_increment
```

Comment:
These might be directly stored as a `Displacement`. It does not bring a lot of value to do so compared to the current state though.

Reply:
This is just a comment, I am OK with the current state, and I let you decide if changing it is worthwhile or not.